pre_install_report/library/pre_install_check_permissions.py

```diff
@@ -42,6 +45,9 @@
 PROVISIONER_AZURE_DISK = "kubernetes.io/azure-disk"
 PROVISIONER_AWS_EBS = "kubernetes.io/aws-ebs"
 
+INGRESS_REL_EQUAL = '==1.18'
```

Comment:
should be >=1.18, we shouldn't fail if they are at 1.19 since we should support it I think

Reply:
@erharb Resolution:
    Kevin and I reviewed the code think we should leave it the way it is because -
1. the old Ingress manifest is specific to 1.18                                                              
2. The new ingress manifest applies  to 1.19 and later.                                                               
3. We do not specify a max version version that we support     
4. Any K8s version less than 1.18 will log an error message - though tool will not fail

Reply:
Thanks for the explanation, then I would recommend renaming the INGRESS_REL_EQUAL property to make it more clear what it's purpose is for the legacy manifest. Perhaps INGRESS_V1BETA1_REL_EQUAL

Reply:
I have renamed the properties with more generic names so that I can just changes the values on the next ingress version change.

INGRESS_MIN_SUPPORTED_REL_EQ = '==1.18'  (Use the previous ingress manifest with beta1 release and format specified )
INGRESS_UNSUPPORTED_REL_LT = '<1.18'  (if this condition met - log an error message)